pages/about/index.md

```diff
@@ -2,22 +2,56 @@
+
+**It was time to come up with something good enough for everyone instead of having every company doing its own battle. We can learn a lot together if we join forces, so we decided to create AsyncAPI!**
+
+## Open governance model
+
+Free software allows users to apply, modify, and continue in the chain by distributing copies to help others and, in this case, disseminating modified versions of the specification. 
+
+Partnership with Postman speeded up the initiative's work in transferring AsyncAPI to a foundation and forming [an open governance model](https://github.com/asyncapi/community/blob/master/CHARTER.md) to assure the community that a single company does not control AsyncAPI Initiative. This move takes the project to a higher level. 
+
+In doing so, we achieve:
+
+- Ensuring an open governance model where no single company dominates the specification and its tools.
+- Making the project as democratic as possible.
+- Relying on an asynchronous decision-making process that empowers the people who "work," not the companies who "pay." 
+
+In other words, it [gives equal power to individual and corporate contributors](/blog/governance-motivation).
 
 ## Maintainers 
 
-AsyncAPI is actively maintained by:
+**Committers** (also known as **Maintainers**) are the decision-makers at the repository/project level. They regularly contribute to the project and are invited by other maintainers to manage the repository, have more privileges, and approve pull requests.
 
-- [Fran Méndez](https://twitter.com/fmvilas) - Project Director.
-- [Łukasz Górnicki](https://twitter.com/derberq) - Maintainer and Dev Community Keeper.
+The committers rule the repository, but for issues that go beyond that, there is a [Technical Steering Committee (TSC)](/community/tsc). This committee comprises all the maintainers who voluntarily want to be part of this committee, thus taking responsibility for overseeing the initiative.
 
-So far, we have had many contributors and we are thankful for their time and help.
+You can see the list of maintainers who are also part of the [AsyncAPI TSC here](/community/tsc).
 
 ## Contributors
 
 We do our best to recognize every contribution to the project. We do it individually in every repository from [AsyncAPI GitHub organization](https://github.com/asyncapi/). There should always be a **Contributors** section in the readme, like [this one](https://github.com/asyncapi/asyncapi/blob/master/README.md#contributors). We use [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. 
 
 We apologize in advance if we failed in recognizing your work. Feel free to contact us on [slack](https://www.asyncapi.com/slack-invite/) and we will fix it immediately, or just talk to [All Contributors](https://allcontributors.org) bot.
```

Comment:
Maybe it is more correct to write _Slack_ with a capital letter at the beginning.